src/components/PlayerComponent.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import PlayerLifeButton from './PlayerLifeButton';
+
+export class PlayerComponent extends Component {
+  state = {
+    currentLife: 40,
+    minLife: 0
+  };
+  addLife = () => {
+    this.setState((prevState, props) => ({
```

Comment:
You can leave `props` out from the parameters if you aren't using it. (Either way, not a big deal!)

Also, you could do `this.setState({currentLife: this.state.currentLife - 1})`